### PR TITLE
update(gossip): support new tpu_vote_quic socket

### PR DESCRIPTION
**Pull request by 0xNineteen**

### CODE REVIEW

**Review by Rexicon226 — CHANGES_REQUESTED** [review]
can you update the "analogous to" links

**Comment by 0xNineteen** [comment]
done - we use Analogous only when we change the name of the struct from agave so we can do forward and backward ctrl+F searches in Sig

**Review by Rexicon226 — APPROVED** [review]
lgtm

File: src/gossip/data.zig [hunk 1]
```diff
@@ -1123,15 +1123,16 @@ pub const SocketTag = enum(u8) {
     tpu_forwards_quic = 7,
     tpu_quic = 8,
     tpu_vote = 9,
-    /// Analogous to [SOCKET_TAG_TVU](https://github.com/anza-xyz/agave/blob/0d34a1a160129c4293dac248e14231e9e773b4ce/gossip/src/contact_info.rs#L36)
+    tpu_vote_quic = 12,
```
Comment (dnut) [inline src/gossip/data.zig:1126]: not a big deal but it's a little weird how this is out of order

```
8
9
12
10
11
```

Comment (0xNineteen) [reply src/gossip/data.zig:1126]: yeah - thats how it is in agave -- makes sense to group tpu_vote and tpu_vote_quic together

**Pull request merged**